### PR TITLE
Contact form  and its validation 

### PR DESCRIPTION
**Pull request by 0sugo**
In this pr I did the following:

Built  a contact form
Added form validations
Added a footer
Modified my readme.md file
Had peer to peer collaboration

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -205,5 +205,31 @@ <h2 class="about-me-topper">About me</h2>
...
-    </body>
-    </html>
\ No newline at end of file
+
+    <section class="form">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form class="contact_form" action="https://formspree.io/f/mayzodlp" method="post">
+            <ul class="form-inputs">
+                <li><input type="text" placeholder="Full name" required maxlength="30" id="name" name="user_name"></li>
+                <li><input type="email" placeholder="Email address" id="email" name="user_email"></li>
```
Comment (pinkmoon25) [inline index.html]: - [x] As mentioned in the project requirements please add the required attribute to email input also, currently it is possible to submit the form without an email, please fix it

File: index.html [hunk 2]
```diff
@@ -205,5 +205,31 @@ <h2 class="about-me-topper">About me</h2>
...
+                <li><textarea id="message" name="user_message" maxlength="30" placeholder="Enter text here"></textarea>
```
Comment (pinkmoon25) [inline index.html]: - [x] As mentioned above please add the required attribute on the textarea element too currently we can submit the form with an empty message
- [x] Also please correct the value of maxlength attribute for the textarea or messages it is supposed to be 500

File: styles.css [hunk 3]
```diff
@@ -383,3 +385,104 @@ li a {
...
+.contact_form {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.form-inputs {
+  padding-left: 0;
+  width: 70%;
+}
```
Comment (pinkmoon25) [inline styles.css]: - [x] Please increase the width of your forms fields to match the figma template, notice how it is taking around 90% of the viewport width in the design :point_down: 
image

your design :point_down: 
image

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @0sugo @Naledi-Dikgale , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark: 
pair programming :heavy_check_mark: 
github flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @0sugo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**